### DISCUSSION TITLE
what can i do if wander reset all store instance state with call reset(or other api) only once

### DISCUSSION THREAD

**Discussion by CHANGEORT** [vuejs/pinia#1728] [Help and Questions]
### What problem is this solving
what can i do if wander reset all store instance state with call reset(or other api) only once
.e.g   when user logout， need reset all store interface. Without call reset func each store.

### Proposed solution

maybe add reset func at ponia root instance

**Comment by wobsoriano** [comment]
https://lobotuerto.com/notes/til-how-to-reset-all-stores-in-pinia
